### ISSUE TITLE
Ticketbooth doesn't run after latest flatpak update

### ISSUE THREAD

**Issue by Scotty-Trees** [aleiepure/ticketbooth#22]
Just updated the latest flatpak update for Ticketbooth (even did a reboot) and the app is unable to launch. Here is the terminal output when I attempt to run it, but it fails. Hope this can be addressed ,thank you!

```
[scott@fedora ~]$ flatpak run me.iepure.Ticketbooth
Traceback (most recent call last):
  File "/app/bin/ticketbooth-bin", line 46, in <module>
    from src import main
  File "/app/share/ticketbooth/src/main.py", line 24, in <module>
    from . import shared  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^
  File "/app/share/ticketbooth/src/shared.py", line 27, in <module>
    faulthandler.enable(file=open(data_dir/'logs'/'crash.log',"w"))
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/scott/.var/app/me.iepure.Ticketbooth/data/logs/crash.log'
[scott@fedora ~]$ 
```

This was just a quick post, if you need more information, I'd be happy to update and provide additional details.

**Comment by aleiepure** [comment]
This is what happens when you don't sleep enough trying to debug crashes... 
Fixed in 5db4928 and (another) update is uploading to Flathub

**Comment by Scotty-Trees** [comment]
No worries, it’s always the little things that get ya sometimes! Thanks very much for the speedy reply and quick fix! And good luck on catching some sheep (sleep) next time☺️
